services-collections.md

```diff
@@ -192,14 +337,23 @@ You may also pass a key / value pair to the `contains` method, which will determ
 
 Finally, you may also pass a callback to the `contains` method to perform your own truth test:
 
-    $collection = new Collection([1, 2, 3, 4, 5]);
+    $collection = collect([1, 2, 3, 4, 5]);
 
-    $collection->contains(function ($key, $value) {
+    $collection->contains(function ($value, $key) {
         return $value > 5;
     });
 
     // false
 
+The `contains` method uses "loose" comparisons when checking item values, meaning a string with an integer value will be considered equal to an integer of the same value. Use the [`containsStrict`](#method-containsstrict) method to filter using "strict" comparisons.
+
+<a name="method-containsstrict"></a>
+#### `containsStrict()` {#collection-method}
+
+This method has the same signature as the [`contains`](#method-contains) method; however, all values are compared using "strict" comparisons.
+
+> {tip} This method's behavior is modified when using [Eloquent Collections](/docs/{{version}}/eloquent-collections#method-contains).
```

Comment:
Please be on the lookout for these Laravel URLs and update them to point to the correct October URL

Reply:
I am, this PR is marked not ready for review, exactly for this reason.